jkube-kit/build/service/buildpacks/src/main/java/org/eclipse/jkube/kit/service/buildpacks/BuildPackBuildOptions.java

```diff
@@ -38,4 +38,5 @@ public class BuildPackBuildOptions {
   private List<String> tags;
   private Map<String, String> env;
   private List<String> volumes;
+  private String appDirPath;
```

Comment:
What's `appDir` in this context? i.e. why are we prepending this prefix if the `pack` CLI doesn't have any notion of this

Reply:
Shall I just rename this field to `path` like pack CLI's `--path` flag?